### ISSUE TITLE
"controlsDir" not used?

### ISSUE THREAD

**Issue by przemekl** [molgenis/CoNVaDING#42]
What does the following warning mean?
"-controlsDir" is directly listed at https://molgenis.gitbooks.io/convading/#cnv-detection
#####
COMMANDLINE OPTIONS IN AFFECT:
-mode StartWithBestScore
-inputDir dir1
-controlsDir dir2
-outputDir dir3
##### WARNING ##### WARNING #####
User specified parameter "controlsDir" not used in best score analysis.

**Comment by ljohansson** [comment]
You are correct. There is no need to use the -controlsDir option, since the file locations are specified in the startWithMatchScore output.
